corehq/apps/hqadmin/migrations/0007_esrestorepillowcheckpoint_datefield.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('hqadmin', '0006_esrestorepillowcheckpoints'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='esrestorepillowcheckpoints',
+            name='date_updated',
+            field=models.DateField(),
+        ),
```

Comment:
If this is already on staging is there a safe way to make it one migration without breaking staging deploys?